src/main/java/reactor/netty/channel/FluxReceive.java

```diff
@@ -340,11 +340,10 @@ final void onInboundComplete() {
 
 	final void onInboundError(Throwable err) {
 		if (isCancelled() || inboundDone) {
-			if (log.isWarnEnabled()) {
-				log.warn(format(channel, "An exception has been observed post termination, use DEBUG level to see the full stack: {}"), err.getClass().getSimpleName());
-			}
-			else if (log.isDebugEnabled()) {
+			if (log.isDebugEnabled()) {
 				log.error(format(channel, "An exception has been observed post termination"), err);
```

Comment:
@smaldini I think we should switch this to `log.debug`. Wdyt?

Reply:
If I may join the discussion: Changing the severity as a consequence of the log level change is somewhat unexpected. IMHO, keeping the level consistent for both branches (e.g., WARN) would come up with the least surprise.

Reply:
agreed